src/components/EventList/index.js

```diff
@@ -0,0 +1,13 @@
+import EventListPreview from "../EventListPreview";
+import StyledListContainer from "../StyledListContainer";
+import { uid } from "uid";
+
+export default function EventList({ events }) {
+  return (
+    <StyledListContainer>
+      {events.map((event) => (
+        <EventListPreview key={uid()} event={event} />
```

Comment:
Never use `uid()` within render/mapping. Your events should include an id property. Use it. And if it is not present, create one since you'll need it.

Reply:
😅 Thanks so much. Better change it now before it's too late.